core/integration/container_test.go

```diff
@@ -3000,26 +3000,23 @@ func TestContainerExecError(t *testing.T) {
 	})
 
 	t.Run("truncates output past a maximum size", func(t *testing.T) {
-		// fill a byte buffer with a string that is twice the size of the max output
+		// fill a byte buffer with a string that is slightly over the size of the max output
 		// size, then base64 encode it
 		var stdoutBuf bytes.Buffer
-		for i := 0; i < 2*core.MaxExecErrorOutputBytes; i++ {
+		for i := 0; i < core.MaxExecErrorOutputBytes+50; i++ {
```

Comment:
Nit: could we explain in a follow-up PR what that `50` is? I always like to capture the meaning of values in a descriptive variable name, comments are easy to miss.

This is an important change, going to merge it straight away.

Reply:
It's explained in the comment above it:

<img width="876" alt="Screenshot 2023-05-25 at 17 46 09" src="https://github.com/dagger/dagger/assets/174525/bd85fd8f-157e-4b73-8172-abb2a3af7fa7">